### PR TITLE
core, history: add None to all builds

### PR DESCRIPTION
**Pull request by amrohassaan**
Add None to all builds that don't have the test in test history so
it shows 'n/a' in test history view. The downside is it adds all the project environments even if they are not relevant.
after

### CODE REVIEW

**Comment by amrohassaan** [comment]
@mwasilew changed this to show either or 'n/a' for all environments in the project.

File: squad/core/history.py [hunk 1]
```diff
@@ -77,14 +77,25 @@ def __init__(self, project, full_test_name, top=None, page=1, per_page=20):
...
 
             results[build][environment] = TestResult(test, suite, metadata, known_issues)
+            remaining_envs = all_envs - set([environment])
+            for env in remaining_envs:
+                results[build][env] = None
+                environments[env] = True
+
+        # Make sure all builds that don't have the test have None at least
+        for b in builds:
+            if not results[build]:
```
Comment (mwasilew) [inline squad/core/history.py:95]: hmm, this is weird, loop variable is 'b' not 'build'. Is this a typo?

**Review by mwasilew — CHANGES_REQUESTED** [review]
Overall idea is OK. Please fix the test errors and it can be merged.

File: squad/core/history.py (hunk 1 above)
Comment (amrohassaan) [reply squad/core/history.py:95]: @mwasilew ouch..yes..typo. thanks for the catch

**Comment by amrohassaan** [comment]
@mwasilew tests uncovered a bug and i fixed it.

**Comment by chaws** [comment]
@amrohassaan is this good to merge?

**Comment by amrohassaan** [comment]
Yes @chaws, it is ready.

**Pull request merged**